src/Native/Silk.NET.OpenAL.Native/Silk.NET.OpenAL.Native.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <SilkSourceLinkExempt>true</SilkSourceLinkExempt>
+  </PropertyGroup>
+
+  <Import Project="..\..\..\build\props\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net4.6.1</TargetFrameworks>
+    <PackageId>Silk.NET.Native.OpenAL</PackageId> 
+    <PackageVersion>2.3.0</PackageVersion>
+    <Authors>.NET Foundation and Contributors</Authors>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackageLicenseExpression>Zlib</PackageLicenseExpression>
```

Comment:
OpenAL Soft is LGPL

Reply:
@HurricanKai - In one of the files I see, 

```
 * This library is free software; you can redistribute it and/or
 *  modify it under the terms of the GNU Library General Public
 *  License as published by the Free Software Foundation; either
 *  version 2 of the License, or (at your option) any later version.
 ```

So I'm assuming we should be using `LGPL-2.0-or-later` for `PackageLicenseExpression`? (https://spdx.org/licenses/)?

Reply:
Sounds right to me, also update this in the readme please